core/src/main/kotlin/fr/sncf/osrd/api/etcs/ETCSBrakingCurvesEndpoint.kt

```diff
@@ -225,6 +223,21 @@ class ETCSBrakingCurvesEndpoint(
         return Envelope.make(*mrspParts.toTypedArray())
     }
 
+    private fun buildETCSConflictCurves(
+        eoaBrakingCurves: EOABrakingCurves
+    ): List<ETCSConflictCurves> {
+        return eoaBrakingCurves.map {
+            assert(it.key.eoaType == EoaType.SPACING || it.key.eoaType == EoaType.ROUTING)
+            ETCSConflictCurves(
+                buildSimpleEnvelope(it.value[BrakingType.IND]?.brakingCurve)!!,
```

Comment:
As it can never be null:
```suggestion
                buildSimpleEnvelope(it.value[BrakingType.IND]!!.brakingCurve)!!,
```